api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
OK, I will have a overview of it.
and to your opinion, which way is more reasonable?